src/static/devika/styles.css

```diff
@@ -0,0 +1,55 @@
+body {
```

Comment:
Can you embed all the css settings in js file? (scripts.js)

Reply:
Also, FYI, use `Visualization 5` section in `./src/static/index.html`
That is where you can put your html ID!